src/utils/generate_structs.jl

```diff
@@ -61,7 +61,7 @@ end
 {{/has_null_values}}
 {{#accessors}}
 {{#create_docstring}}\"\"\"Get {{struct_name}} {{name}}.\"\"\"{{/create_docstring}}
-{{accessor}}(value::{{struct_name}}) = value.{{name}}
+{{accessor}}(value::{{struct_name}}) = {{#needs_conversion}}get_value({{data_type}}, value, :{{name}}){{/needs_conversion}}{{^needs_conversion}}value.{{name}}{{/needs_conversion}}
```

Comment:
I suggest that this would be a better end result of generated code.
```
get_active_power(value::ThermalStandard)= get_value(value, value.active_power)
```
Dispatch would work on the type of the field and you wouldn't have to include it explicitly.

Reply:
I feel is the same thing, but sure I can simplify it.

Reply:
It's very possible that I am misunderstanding the complete flow.  I was expecting to see something like this:

```
# Auto-generated
get_active_power(value::ThermalStandard) = convert_power(value, value.active_power)
```

```
function convert_power(c::Component, value::Float64)
    units_settings = get_units_settings(c)
    # do math on value
    return converted_value
end

function convert_power(c::Component, value::MinMax)
    units_settings = get_units_settings(c)
    # do math on value
    return converted_value
end
```

Can you point me to an example of what you were thinking?

Reply:
We should discuss this to reach a resolution.

Reply:
What else is left to discuss?